examples_tests/28.BSDFValidator/Main.cpp

```diff
@@ -18,7 +18,7 @@ int main()
     params.Bits = 24; //may have to set to 32bit for some platforms
     params.ZBufferBits = 24; //we'd like 32bit here
     params.DriverType = video::EDT_OPENGL; //! Only Well functioning driver, software renderer left for sake of 2D image drawing
-    params.WindowSize = core::dimension2d<uint32_t>(1280, 720);
+    params.WindowSize = core::dimension2d<uint32_t>(512, 512);
```

Comment:
you will probably want to display the buttons/UI not over the texture, so you need a larger window